packages/mobile/src/backup/BackupForceScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import Button from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { BackHandler, ScrollView, StyleSheet, Text } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { OnboardingEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import DelayButton from 'src/backup/DelayButton'
+import { Namespaces } from 'src/i18n'
+import Logo from 'src/icons/Logo'
+import { emptyHeader } from 'src/navigator/Headers'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+
+type Props = StackScreenProps<StackParamList, Screens.BackupForceScreen>
+
+function BackupForceScreen({ navigation }: Props) {
+  const { t } = useTranslation(Namespaces.backupKeyFlow6)
+
+  const startBackup = () => {
+    ValoraAnalytics.track(OnboardingEvents.backup_start)
+    navigate(Screens.AccountKeyEducation)
+  }
+
+  // Prevent back button on Android
+  useEffect(() => {
+    const backPressListener = () => true
+    BackHandler.addEventListener('hardwareBackPress', backPressListener)
+    return () => BackHandler.removeEventListener('hardwareBackPress', backPressListener)
+  }, [])
```

Comment:
We also need to prevent swipe back on iOS.

https://reactnavigation.org/docs/preventing-going-back/

Reply:
We are, the navigation options have a `gestureEnabled: false,` that prevents it.

I tried using the `beforeRemove` above but it didn't really prevent the swiping, it just reverted it. Like I swiped and after I lifted my finger from the screen it swiped back to the original position. The result was the same but it looked a bit strange imo. The `gestureEnabled: false` just prevents swiping in the first place.

Reply:
Right, missed that, what you did has much better UX 💯